misk-hibernate-testing/src/main/kotlin/misk/hibernate/HibernateTestingModule.kt

```diff
@@ -34,7 +34,8 @@ class HibernateTestingModule(
   private val qualifier: KClass<out Annotation>,
   private val config: DataSourceConfig? = null,
   private val startUpStatements: List<String> = listOf(),
-  private val shutDownStatements: List<String> = listOf()
+  private val shutDownStatements: List<String> = listOf(),
+  private val enableScaleSafetyChecks: Boolean = true
```

Comment:
Wanna flip the default? These are broken and should be off til they're fixed.

Reply:
Makes sense; will do!